core/src/apps/monero/live_refresh.py

```diff
@@ -49,10 +49,10 @@ def __init__(self):
         ctx, misc.validate_full_path, keychain, msg.address_n, CURVE
     )
 
-    passphrase_fprint = get_passphrase_fprint()
-    if live_refresh_token() != passphrase_fprint:
+    fingerprint = storage.cache.get_session_id()[:4]
```

Comment:
this is doing something very different from the original code
namely: in the original code, you could get `passphrase_fprint` by knowing the passphrase + device_id; i.e., it was possible to live-refresh on a cold-started device

now you need to know the current session_id, but not the passphrase

i didn't look into Monero code deep enough to know what this means, so making a note here

Reply:
The session_id is always coupled to a single passphrase. I've talked to @ph4r05 before briefly and he mentioned he needs just a simple identifier of the session and he seemed fine with this solution. But let's cc him on this to make double sure.

Reply:
@tsusanka correct, I just need some session ID (which changes per `(Trezor boot, passphrase)`) so I don't have to annoy users by asking for confirmation every minute.

Reply:
interesting
this highlights a usecase for something like `storage.cache.set(key, value)` that would bind the key-value pair to the session, and would go away with ClearSession or with passphrase change.

as it happens Cardano's `_cached_root` could also use that, and I'm afraid the current implementation is either incorrect, or only works through magic (see other comment)